.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
Please also enable the container based-infrastructure. (http://docs.travis-ci.com/user/workers/container-based-infrastructure/)